### PR TITLE
[MAINT] enact deprecations due for 0.11.0

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Closes none but relates to #3812

Changes proposed in this pull request:

- remove deprecated functions
- change default value of deprecated parameters

### CODE REVIEW

**Review by bthirion — APPROVED** [review]
LGTM, thx !

**Comment by Remi-Gau** [comment]
Will wait until we are a bit closer to release date before merging.

**Pull request merged**
